test/Table_Tests/src/Util.enso

```diff
@@ -8,12 +8,6 @@ import Standard.Test.Extensions
 polyglot java import org.enso.base_test_helpers.FileSystemHelper
 
 Table.should_equal self expected =
```

Comment:
If we are improving this, I think it's useful to also add a typecheck - then if by accident we get a wrong type here, we will get a clear message instead of a panic like `Unresolved_Symbol <columns>`.

```suggestion
Table.should_equal self expected = case expected of
    _ : Table -> ...
    _ -> Test.fail "Got a Table, but expected a "+expected.to_display_text
```